### DISCUSSION TITLE
What is the maximum number of email messages you can send from a single SMTP connection in Postal?

### DISCUSSION THREAD

**Discussion by bukunmilab** [postalserver/postal#3183] [Help with using Postal]
Hey guys,
So what is the average or maximum email messages that can be sent from a single SMTP connection in Postal?
Can one SMTP connection send out 10,000 emails before the connection gets closed?

I need the maximum figure possible as I working on Mailwizz configuration based on this.

**Comment by willpower232** [comment]
It depends how Mailwizz manages the connection, presuming it is using SMTP and using the reset command between each message, it could go on for a while but this sounds like a micro optimisation that won't really help you too much in my opinion.
